bib/cmd/bootc-image-builder/main.go

```diff
@@ -185,10 +185,27 @@ func manifestFromCobra(cmd *cobra.Command, args []string) ([]byte, *mTLSConfig,
 	imgTypes, _ := cmd.Flags().GetStringArray("type")
 	rpmCacheRoot, _ := cmd.Flags().GetString("rpmmd")
 	targetArch, _ := cmd.Flags().GetString("target-arch")
-	tlsVerify, _ := cmd.Flags().GetBool("tls-verify")
-	localStorage, _ := cmd.Flags().GetBool("local")
 	rootFs, _ := cmd.Flags().GetString("rootfs")
 
+	// If --local was given, warn in the case of --local or --local=true (true is the default), error in the case of --local=false
+	if cmd.Flags().Changed("local") {
+		localStorage, _ := cmd.Flags().GetBool("local")
+		if localStorage {
+			fmt.Fprintf(os.Stderr, "WARNING: --local is now the default behavior, you can remove it from the command line\n")
+		} else {
+			return nil, nil, fmt.Errorf(`--local=false is no longer supported, remove it and make sure to pull the container before running bib:
+	sudo podman pull %s`, imgref)
+		}
+	}
+
+	// --local is the default now, so there is no longer a need for --tls-verify
+	if cmd.Flags().Changed("local") {
```

Comment:
silly question, why do we need to check "local" here, could we just skip that and check for only for that?

Reply:
I'll ping @ondrejbudai here because this was his change. I think the idea was to catch when `--local=false` was set and error out on it.

Reply:
The "local" case *seems* to be covered in the lines above, this is why I'm wondering. But maybe I'm just missing something, I only had a casual look at this part :)

Reply:
Typo again 🤦 this should be `tls-verify`